src/status_im2/contexts/add_new_contact/views.cljs

```diff
@@ -33,13 +36,20 @@
        [quo/text (style/text-description)
         (i18n/label :t/ens-or-chat-key)]
        [rn/view style/container-text-input
-        [rn/text-input
-         (merge (style/text-input error?)
-                {:default-value  input
-                 :placeholder    (i18n/label :t/type-some-chat-key)
-                 :on-change-text #(debounce/debounce-and-dispatch
-                                   [:contacts/set-new-identity %]
-                                   600)})]
+        [rn/view (style/text-input-container error?)
+         [rn/text-input
```

Comment:
curious, do we not have a text input from quo2? 

Seems like this should get implemented soon as I assume we should be using this here instead of rn/text-input.

Reply:
We have it but it's styled differently. Technically we can migrate to quo/text-input and use it here but I think it's a bit out of scope.

Reply:
hmm, we should not be using `quo` components for anything new afaik. Either way yes it's out of scope for this work